### PR TITLE
Menu functionality + all buttons work (except YouTube one)

### PR DESCRIPTION
**Pull request by RalucaRbc**

### CODE REVIEW

**Review by M00N89 — COMMENTED** [review]
cred ca iar am apasat pe ceva ce nu trebuia :))

**Review by M00N89 — APPROVED** [review]

**Review by BogdanNegura — APPROVED** [review]
💯 Nice "bogdan.png" :)

**Review by biggicode — APPROVED** [review]

File: src/_component/open-source/open-source.scss [hunk 1]
```diff
@@ -66,13 +66,16 @@
...
   }
 
-  span[class$="TypeScript"] {
+  span[class$="SCSS"] {
     background-color: #20D55D;
   }
 
   span[class$="CSS"] {
     background-color: #D52027;
   }
```
Comment (MariaTeo) [inline src/_component/open-source/open-source.scss:75]: You could add an empty line for readability

**Review by MariaTeo — COMMENTED** [review]
Nice work

**Review by MariaTeo — APPROVED** [review]

**Review by Constantin-Matei — APPROVED** [review]
:)

**Pull request merged**